src/System.Buffers.Primitives/System/Buffers/OwnedBuffer.cs

```diff
@@ -69,7 +40,11 @@ public void Dispose()
         public abstract void Retain();
 
         public abstract void Release();
-
         #endregion
+
+        protected static unsafe void* Add(void* pointer, int offset)
```

Comment:
Is adding it to [S.R.CS.U](https://github.com/dotnet/corefx/blob/master/src/System.Runtime.CompilerServices.Unsafe/ref/System.Runtime.CompilerServices.Unsafe.cs) a valid option?

cc @jkotas